models/topics.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection.js");
+const utils = require("../db/seeds/utils.js");
```

Comment:
What's this line for atm? Any code currently not being used in this endpoint shouldn't be included yet.